### ISSUE TITLE
The autoscroll feature is not perfect

### ISSUE THREAD

**Issue by Dazaisan99** [Taliayaya/Apando#64]
When switching lobbies?(salons), the autoscrolling goes to the bottom but stops just before the end

![Screenshot_7](https://user-images.githubusercontent.com/96884464/165989281-b3a7f4f1-485e-4da3-aaf1-f4632b6b6ca8.png)

**Comment by Taliayaya** [comment]
Indeed. Uploaded files can take some times before being completely loaded. 
However, the autoscroll often scrolls before that, which means once the files are loaded, it takes more space between messages, and the autoscroll won't take effect.

A solution might be to scroll when files are finished loading ?
